examples/text_prediction/README.md

```diff
@@ -1,15 +1,144 @@
+```
+
+Once the script is finished, you will see a `submission.xlsx` file generate in the 
+`ag_data_scientist_salary` folder and you can try to submit that to the competition leaderboard.
+
+!IMPORTANT. Try to run the experiment on a p3.2x instance :).
+
+## Reach Top-5 Performance in Mercari Price Suggestion
+
+Here, we provide the example that shows how to use AutoGluon to achieve top-5 performance in
+ [Mercari Price Suggestion](https://www.kaggle.com/c/mercari-price-suggestion-challenge/data).
+To run the example, you will need to configure the Kaggle API which will be documented in 
+https://github.com/Kaggle/kaggle-api and download the dataset.
+
+```
+sudo apt install -y p7zip-full
+bash prepare_mercari_kaggle.sh
+```
+
+After you have prepared the dataset, you can use the following command:
+```
+mkdir -p ag_mercari_price_single
+python3 run_competition.py --train_file mercari_price/train.tsv \
+                           --test_file mercari_price/test_stg2.tsv \
+                           --sample_submission mercari_price/sample_submission_stg2.csv \
+                           --task mercari_price \
+                           --eval_metric r2 \
+                           --exp_dir ag_mercari_price_single \
+                           --mode single 2>&1  | tee -a ag_mercari_price_single/log.txt
+```
+
+In addition, you may run multimodal with weighted ensemble
+```
+mkdir -p ag_mercari_price_weighted
+python3 run_competition.py --train_file mercari_price/train.tsv \
+                           --test_file mercari_price/test_stg2.tsv \
+                           --sample_submission mercari_price/sample_submission_stg2.csv \
+                           --task mercari_price \
+                           --eval_metric r2 \
+                           --exp_dir ag_mercari_price_weighted \
+                           --mode weighted 2>&1  | tee -a ag_mercari_price_weighted/log.txt
+```
+Or stacking
+```
+mkdir -p ag_mercari_price_stacking
+python3 run_competition.py --train_file mercari_price/train.tsv \
+                           --test_file mercari_price/test_stg2.tsv \
+                           --sample_submission mercari_price/sample_submission_stg2.csv \
+                           --task mercari_price \
+                           --eval_metric r2 \
+                           --exp_dir ag_mercari_price_stacking \
+                           --mode stacking 2>&1  | tee -a ag_mercari_price_stacking/log.txt
+```
+
+## Solve GLUE Tasks with AutoGluon Text
+
+Here, we show how you may use AutoGluon Text to solve all tasks in the [GLUE benchmark](https://openreview.net/pdf?id=rJ4km2R5t7).
+ 
 ### Prepare the data
 ```bash
 python3 prepare_glue.py --benchmark glue
 ```
 
 ### Run the benchmark
-Run on all datasets
+Run on all datasets with either a single `TextPredictor` model or the `multimodal` configuration 
+in AutoGluon Tabular that will use the `TextPredictor` model in the 5-fold-1-layer stacking.
```

Comment:
"use the `TextPredictor` model in the 5-fold-1-layer stacking."
-> 
"combine the `TextPredictor` model with tabular models from AutoGluon-Tabular via a single layer of  stack ensembling with 5-fold bagging."